monkey/infection_monkey/system_info/windows_cred_collector/windows_credential.py

```diff
@@ -0,0 +1,15 @@
+from typing import Dict
+
+
+class WindowsCredential:
```

Comment:
Credentials in considered a mass noun, so I think this class should be renamed to `WindowsCredentialsSet` (and `WindowsCredentialCollector` -> `WindowsCredential**s**Collector`)

Reply:
Hmmm... Credential (singular) is a valid word (not like scissors for example). In terms of English, I agree. In terms of programming, a class or an object should be in singular form, plural implies an iterable.